jobs/check_for_spam_users.rb

```diff
@@ -0,0 +1,14 @@
+module Jobs
+  class CheckForSpamUsers < ::Jobs::Scheduled
+    every 10.minutes
+
+    def execute(args = {})
+      return unless SiteSetting.akismet_enabled?
+      return if SiteSetting.akismet_api_key.blank?
+
+      DiscourseAkismet::CheckSpamUser.to_check.each do |user|
```

Comment:
```suggestion
      DiscourseAkismet::CheckSpamUser.to_check.find_each do |user|
```
If there are alot of users to check, especially on the first run, we don't want to be loading all the users into memory at once.